apps/src/templates/teacherDashboard/LoginTypePicker.jsx

```diff
@@ -38,11 +35,19 @@ class LoginTypePicker extends Component {
   };
 
   render() {
-    const {title, provider, setLoginType, handleCancel} = this.props;
+    const {
+      title,
+      provider,
+      setLoginType,
+      handleImportOpen,
+      handleCancel,
+      disabled,
+    } = this.props;
     const withGoogle = provider === OAuthSectionTypes.google_classroom;
     const withMicrosoft = provider === OAuthSectionTypes.microsoft_classroom;
     const withClever = provider === OAuthSectionTypes.clever;
-    const anyImportOptions = withGoogle || withMicrosoft || withClever;
+    const anyImportOptions = withGoogle || withMicrosoft || withClever &&
+      (typeof handleImportOpen === 'function');
```

Comment:
Should this actually be:
```(withGoogle || withMicrosoft || withClever) && (typeof handleImportOpen === 'function')```

or do we only care about handleImportOpen for withClever